From 19fa6de891e2ecf8ae8dd7ede5372a17409494f9 Mon Sep 17 00:00:00 2001
From: Alexandre Torgue <alexandre.torgue@st.com>
Date: Thu, 29 Aug 2019 09:20:00 +0200
Subject: [PATCH 23/52] ARM: dts: stm32: use exti 19 as main interrupt to
 support RTC wakeup on stm32mp157

Link between GIC and exti line is now done inside EXTI driver. So in order
to be wake up source exti irqchip has to be used.

Upstream-Status: Denied Downstream stm32-gh v5.4-stm32mp [93a67bfcf197ffad8d8962369b56b8b714d1447e]
Signed-off-by: Alexandre Torgue <alexandre.torgue@st.com>
Change-Id: I70122aaaba01c7553a03f046df7a0d5a8b9b4277
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/142476
Reviewed-by: CITOOLS <smet-aci-reviews@lists.codex.cro.st.com>
(cherry picked from commit 29da5624cdcfd79da8ab3fce409e9b1b97066523)
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/146661
---
 arch/arm/boot/dts/stm32mp151.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/stm32mp151.dtsi b/arch/arm/boot/dts/stm32mp151.dtsi
index 317632c8479c..4b7e1103d87c 100644
--- a/arch/arm/boot/dts/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/stm32mp151.dtsi
@@ -1539,7 +1539,7 @@ rtc: rtc@5c004000 {
 			reg = <0x5c004000 0x400>;
 			clocks = <&rcc RTCAPB>, <&rcc RTC>;
 			clock-names = "pclk", "rtc_ck";
-			interrupts = <GIC_SPI 3 IRQ_TYPE_LEVEL_HIGH>;
+			interrupts-extended = <&exti 19 IRQ_TYPE_LEVEL_HIGH>;
 			status = "disabled";
 		};
 
-- 
2.29.2

